src/pages/_document.js

```diff
@@ -3,7 +3,13 @@ import { Html, Head, Main, NextScript } from "next/document";
 export default function Document() {
   return (
     <Html lang="en">
-      <Head />
+      <Head>
+        <title>Chaotic PPR | Pacstall</title>
+        <meta name="description" content="The (chaotic) Pacstall Package Repository" />
```

Comment:
```suggestion
        <meta name="description" content="The Chaotic Pacstall Package Repository" />
```

Chaotic is part of the name.